### ISSUE TITLE
Bad packet id 83 - 5.0.0 dev

### ISSUE THREAD

**Issue by crafter23456** [dmulloy2/ProtocolLib#1531]
**Describe the bug**
```
[12:25:08] [Server thread/INFO]: [Matrix] (Via) Player crafter2345 joined with (1.18.x/758) client
[12:25:10] [Server thread/INFO]: crafter2345 lost connection: Internal Exception: io.netty.handler.codec.DecoderException: java.io.IOException: Bad packet id 83
[12:25:10] [Server thread/INFO]: crafter2345 left the game.
```

**To Reproduce**
1. Use the 5.0.0 dev build
2. Join a 1.8.8 server
3. Look into the log

**Expected behavior**
That I can join the server like on 4.8.0 :D

**Version Info**
https://paste.labymod.net/qiwabaviwi.cs

**Comment by derklaro** [comment]
Hey, can you try to reproduce the issue without ViaVersion? Tested it on paper 1.8 and had no issues...

**Comment by derklaro** [comment]
Okay checked it and it is a ViaVersion thing. Via relies on the name of the decoder being `protocol_lib_decoder` when the plugin is enabled which is no longer the case. That causes Via to add the decompressor at the wrong position in the pipeline. Removing the  check fixes the problem.
Will move the issue over to ViaVersion and see what they say!

**Comment by derklaro** [comment]
Will be fixed with ViaVersion 4.2.2
